interface/src/main.cpp

```diff
@@ -66,59 +66,219 @@ int main(int argc, const char* argv[]) {
+    QCommandLineOption serverContentPathOption(
+        "serverContentPath",
+        "Path to find server content.", // What content??
+        "serverContentPath"
+    ); // This data type will not be familiar to users.
+    QCommandLineOption overrideAppLocalDataPathOption(
+        "cache",
+        "Set test cache.",
+        "dir"
+    );
+    QCommandLineOption scriptsOption(
+        "scripts",
+        "Set path for defaultScripts. These are probably scripts that run automatically.",
+        "dir"
+    );
+    QCommandLineOption allowMultipleInstancesOption(
+        "allowMultipleInstances",
+        "Allow multiple instances to run."
+    );
+    QCommandLineOption displaysOption(
+        "display",
+        "Preferred display.",
+        "displays"
+    );
+    QCommandLineOption disableDisplaysOption(
+        "disable-displays",
+        "Displays to disable.",
+        "string"
+    );
+    QCommandLineOption disableInputsOption(
+        "disable-inputs",
+        "Inputs to disable.",
+        "string"
+    );
+    QCommandLineOption suppressSettingsResetOption(
+        "suppress-settings-reset",
+        "Suppress the prompt to reset interface settings."
+    );
+    QCommandLineOption oculusStoreOption(
+        "oculus-store",
+        "Let the Oculus plugin know if interface was run from the Oculus Store."
+    );
+    QCommandLineOption standaloneOption(
+        "standalone",
+        "Emulate a standalone device."
+    );
+    QCommandLineOption disableWatchdogOption(
+        "disableWatchdog",
+        "Disable the watchdog thread. The interface will crash on deadlocks."
+    );
+    QCommandLineOption systemCursorOption(
+        "system-cursor",
+        "Probably prevents changing the cursor when application has focus."
+    );
+    QCommandLineOption concurrentDownloadsOption(
+        "concurrent-downloads",
+        "Maximum concurrent resource downloads. Default is 16, except for Android where it is 4.",
+        "integer"
+    );
+    QCommandLineOption avatarURLOption(
+        "avatarURL",
+        "Override the avatar U.R.L.",
+        "url"
+    );
+    QCommandLineOption replaceAvatarURLOption(
+        "replace-avatar-url",
+        "Replaces the avatar U.R.L. When used with --avatarURL, this takes precedence.",
+        "url"
+    );
+    QCommandLineOption setBookmarkOption(
+        "setBookmark",
+        "Set bookmark as key=value pair. Including the '=' symbol in either string is unsupported.",
+        "string"
+    );
+    QCommandLineOption forceCrashReportingOption(
+        "forceCrashReporting",
+        "Force crash reporting to initialize."
+    );
+    // The documented "--disable-lod" does not seem to exist.
+    // Below are undocumented.
+    QCommandLineOption noLauncherOption(
+       "no-launcher",
+       "Do not execute the launcher."
+    );
+    QCommandLineOption overrideScriptsPathOption(
+       "overrideScriptsPath",
+       "Probably specifies where to look for scripts.",
```

Comment:
Remove "probably". This is the path from which scripts are loaded by default. The default is the scripts directory that can be found in the root of the source tree.

Reply:
Can you clarify what "where scripts are loaded by default" means? How is this different from the other script-related parameters?

Reply:
This is the directory where the code will look for scripts.

`--defaultScriptsOverride` is a filename, it's used to replace `defaultsScripts.js`. That's the script that will be loaded automatically on start, and which is used to set up the UI and other basic functionality.

`--scripts` seems broken from the looks of it, and may not be doing anything useful